configure.ac

```diff
@@ -622,6 +622,18 @@ case $host in
      use_gl=no
      USE_STATIC_FFMPEG=1
      ;;
+  aarch64*-*-linux-gnu*|aarch64*-*-linux-uclibc*)
+     target_platform=target_linux
+     ARCH="aarch64"
+     use_arch="aarch64"
+     use_cpu=cortex-a53
```

Comment:
The use_cpu should be dropped, kodi builds & runs fine without it on a dragonboard-410c. Which incidentally works best with GL, not with GLES.